app/src/main/java/org/covidwatch/android/data/backend/ContactEventFetcher.kt

```diff
@@ -0,0 +1,20 @@
+package org.covidwatch.android.data.backend
+
+import java.util.*
+
+/**
+ * Asynchronously fetch contact events from the backend.
+ */
+interface ContactEventFetcher {
+
+    /**
+     * Asynchronously fetch contact events from the backend.
+     * @param timeWindow The time window to fetch the contact events.
+     * @param resultCb Callback for receiving the contact events with the given IDs with the given
+     * infection state. May be called more than once per method call.
+     * @return The resulting contact events.
+     * @exception Exception Throws an exception on error.
```

Comment:
nit: you can specify exactly what can of exceptions might be throws in `@Throws(...)`

Reply:
or even better, you can create callback interface with two methods: `onResult(...)`, `onError(..)`